experiment_buddy/experiment_buddy.py

```diff
@@ -297,46 +296,45 @@ def log_cmd(cmd, retr):
 def _commit_and_sendjob(hostname: str, experiment_id: str, sweep_yaml: str, git_repo: git.Repo, project_name: str,
                         proc_num: int, extra_slurm_header: str, wandb_kwargs: dict):
     git_url = git_repo.remotes[0].url
-    with concurrent.futures.ThreadPoolExecutor(max_workers=1) as executor:
-        scripts_folder = executor.submit(_ensure_scripts, hostname, extra_slurm_header, git_repo.working_dir)
-        hash_commit = git_sync(experiment_id, git_repo)
+    ssh_session = _open_ssh_session(hostname)
+    scripts_folder = ensure_scripts(ssh_session, extra_slurm_header, git_repo.working_dir)
```

Comment:
i was expecting async  here, what happened?

Reply:
maybe make another PR like "parallel processing homogenization" where you turn everything into async

Reply:
I was trying to use an async ssh client, but, given that the majority of the time (>90%) was spent in a single line, the complexity was too high compared to the expected return, a simple sequential run is simpler and the measured time is almost the same

I'll make another PR when I can find some process that could improve the time with a concurrent execution

Reply:
Less complexity<3
I added that to reduce the deployment time by quite a bit, maybe things changed
We really need to measure timing

Reply:
Now we can! :D

Reply:
Moved to another PR